### PR TITLE
Feat/pm 753 matches tab

### PR DESCRIPTION
**Pull request by dejwi**
[PM-753](https://playmakerpro.atlassian.net/browse/PM-753)

### Task Description

- added resuable animated tab loaders (+added it to career)
- added dummy lists api/hooks
- added matches tab

### Additional Notes (optional)

filters list is not scrollable because there are some problems related to (correct) select menu not exisiting, we're waiting for when ui/ux completes it

this table was a hell to make

### CODE REVIEW

File: components/common/Select/select.tsx [hunk 1]
```diff
@@ -78,13 +79,17 @@ function Select({
...
                 {items.find(item => item.value === value)?.name || name}
               </span>
-              <ChevronDownIcon className="h-5 w-5 fill-[inherit] ui-open:hidden" />
-              <ChevronUpIcon className="h-5 w-5 fill-[inherit] ui-not-open:hidden" />
+              <ChevronDownIcon
+                className={clsx('h-5 w-5 fill-[inherit]', open && 'hidden')}
+              />
+              <ChevronUpIcon
+                className={clsx('h-5 w-5 fill-[inherit]', !open && 'hidden')}
+              />
```
Comment (mati-1) [inline components/common/Select/select.tsx:92]: just toggle rotate-180 bro :)

<ChevronDownIcon className={clsx('h-5 w-5 fill-[inherit]', open && 'rotate-180')}  />

File: components/profile/common/tab-loader/tab-wrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use client'
+
+import { motion } from 'framer-motion'
+import React from 'react'
+
+interface IProps {
+  children: React.ReactNode
```
Comment (mati-1) [inline components/profile/common/tab-loader/tab-wrapper.tsx:7]: make it simpler, just import the type and put them in the children

File: components/profile/matches/matches-panel.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+const MatchesPanel = (props: IProfileMatchesSeason) => {
+  const { season, total } = props
+
+  return (
+    <div className="flex flex-col border-t border-darkShade-5 bg-white pb-4">
+      <div className="pt-7">
+        <h4
+          className={seasonHeadingCva({
+            season: season.split(' ')[0] as any,
```
Comment (mati-1) [inline components/profile/matches/matches-panel.tsx:29]: hmm, maybe use unknown instead any? (im not sure)

File: components/profile/matches/matches-panel.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        Lato: '',
+        Jesień: 'text-orangeShade-50',
+        Zima: '',
+      },
+    },
+  },
+)
+
+const MatchesPanel = (props: IProfileMatchesSeason) => {
+  const { season, total } = props
```
Comment (mati-1) [inline components/profile/matches/matches-panel.tsx:22]: why didnt you destructure it? 🤔

File: components/profile/matches/table/components.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      className="icon-16"
+    />
+    <span className="max-w-[6.2rem] flex-1 truncate pl-[0.375rem] pr-3 text-body-xs text-darkAlpha-40">
+      {team.name}
+    </span>
+    <span className="w-8 text-center text-body-sm font-bold text-darkAlpha-40">
+      {team.score}
+    </span>
+  </div>
+)
```
Comment (mati-1) [inline components/profile/matches/table/components.tsx:90]: wouldnt a component folder be better? 🤔

**Review by mati-1 — COMMENTED** [review]
bro your code is really good ⚡ , but tell me about my questions 💯

File: components/common/Select/select.tsx (hunk 1 above)
Comment (dejwi) [reply components/common/Select/select.tsx]: you're right 😅

File: components/profile/common/tab-loader/tab-wrapper.tsx (hunk 2 above)
Comment (dejwi) [reply components/profile/common/tab-loader/tab-wrapper.tsx]: oh, this import is unnecessary, probably forgot to remove it as I create react boilerplate on each new file with `rafce` and it adds this react import at top
removed it

File: components/profile/matches/matches-panel.tsx (hunk 3 above)
Comment (dejwi) [reply components/profile/matches/matches-panel.tsx:29]: there's a error then

Comment (mati-1) [reply components/profile/matches/matches-panel.tsx:29]: ok ok i dont remember when to use unknown right

**Review by mati-1 — APPROVED** [review]
💯 / 💯

**Pull request merged**